osu.Game/Skinning/GlobalSkinnableContainerLookup.cs

```diff
@@ -15,33 +15,33 @@ public class GlobalSkinnableContainerLookup : ISkinComponentLookup, IEquatable<G
         /// <summary>
         /// The target area / layer of the game for which skin components will be returned.
         /// </summary>
-        public readonly GlobalSkinnableContainers Target;
+        public readonly GlobalSkinnableContainers Component;
```

Comment:
Not sure of this rename, I'd probably propose to call this something like `GlobalSkinnableContainer` or `SkinnableContainer` or something.

Unless I'm *wildly* misunderstanding and about to open another debate into naming (in which case please do feel free to immediately tell me to shut the ---- up), `GlobalSkinnableContainerLookup` is supposed to lookup skinnable containers, which are _not_ individual skinnable _components_ themselves (think score counter, health bar, etc.), but rather _containers of skinnable components_.

If that understanding is correct then I find this being called `Component` misleading because _it's not going to look up a component as I think of it_.

Reply:
As for my perspective, I originally wanted this to be called `GlobalSkinComponentLookup`, because I think of everything skinning as a "component". In that case, this being `Component` makes sense to me.

In my original PR, I named this `Target` so I'm definitely thinking in the direction of renaming.

I disagree with `GlobalSkinnableContainer` or the other suggestion because there is a world where this is moved to `ISkinComponentLookup` as I originally had, at which point being called that doesn't really make sense. I'm not really sure what path this will go down in the end, but it feels too inflexible and hard-lined as that.

How about `Lookup`? We are throwing this around in the office and it seems agreeable.

Reply:
> As for my perspective, I originally wanted this to be called `GlobalSkinComponentLookup`, because I think of everything skinning as a "component". In that case, this being `Component` makes sense to me.

If this is the consensus from everyone I'll just rewire my brain. `Lookup` is also fine I guess.

Reply:
I think I'm alone in that regard 😛